data/data-pipeline/data_pipeline/etl/runner.py

```diff
@@ -32,36 +33,62 @@ def get_datasets_to_run(dataset_to_run: str):
     return dataset_list
 
 
-def etl_runner(dataset_to_run: str = None) -> None:
-    """Runs all etl processes or a specific one
+def build_instance_class_from_dataset(dataset: str = None) -> None:
```

Comment:
This function expects a single dataset or all by default. I have just modularized this to handle one ETL dataset. I am wondering if in the processing logic we should be checking that `dataset_list` for length? We already validate the dataset.

Reply:
Thanks @sverchdotgov. I will address this and change the name.